### PR TITLE
added blogpost "Why Kubeflow in your infrastructure?"

### PR DESCRIPTION
**Pull request by netankit**
Kubeflow Community-PM group contribution
Refer: https://groups.google.com/forum/#!topic/kubeflow-pm/LzlWu2LCveY

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/289)

### CODE REVIEW

File: content/blog/why_kubeflow.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
++++
+title = "Why Kubeflow in your Infrastructure?"
+description = "Answering the 'why' about Kubeflow in an organization's Infrastructure."
+weight = 20
+publishDate = 2018-11-09T02:00:00+01:00
+draft = false
++++
+
+##### Guest post by Ankit Bahuguna & Faheem Nadeem, Software Engineer(s), Cliqz GmbH
```
Comment (abhi-g) [inline content/blog/why_kubeflow.md]: Generally, Kubeflow blog posts present author attribution at the end of the post.

File: content/blog/why_kubeflow.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+![Kubeflow](../kf.png)
```
[the remark this answers is not in the document]
Comment (netankit) [reply content/blog/why_kubeflow.md]: @abhi-g  Thanks for the review. I have updated the PR based on review comments in e1ec0d8.

**Pull request merged**